Shared/Prefs.swift

```diff
@@ -140,6 +142,9 @@ public struct PrefsKeys {
     // The last timestamp we polled FxA for missing send tabs
     public static let PollCommandsTimestamp = "PollCommandsTimestamp"
 
+    // The last recorded timestamp
+    public static let LastRecordedTimestamp = "LastRecordedTimestamp"
```

Comment:
This pref needs a better name or else we don't know the context it will be used without checking the code. Maybe `FakespotLastCFRTimestamp` would be better?